views/index.erb

```diff
@@ -0,0 +1,6 @@
+<h1> Please register your player name: </h1>
```

Comment:
To improve the UI maybe add images and background colour. https://www.w3schools.com/html/tryit.asp?filename=tryhtml_styles_background-color but overall clear layout and easy for user to understand what to do.